### ISSUE TITLE
[DESIGN] Timeseries - Locations

### ISSUE THREAD

**Issue by LirryPinter** [nens/lizard-management-client#588]
- Design list
- Design form

**Comment by LirryPinter** [comment]
Design
https://xd.adobe.com/view/64bdf742-fee5-4907-a0f1-e9f27ecac3fc-31ee/

**Comment by lexvand** [comment]
@LirryPinter 
I did some research on the (im)possibilities of finding the asset you want to relate a location to. My conclusions:
- Asset endpoints in API v3 do not support this, as filtering on name is not supported
- Asset endpoints in API v4 are still to be made, but question is whether we can support this type of filtering there. We have discussed this before and conclusion was that this kind of filtering (icontains on name) on these tables is too heavy.
- The option we have left is the search endpoint in v3. The options for searching work quite well, just a few limitations (e.g. nested assets not supported). Have a look at the documentation as described on the endpoint.
- An example request: https://nxt3.staging.lizard.net/api/v3/search/?q=purmerend&type=measuringstation
